### ISSUE TITLE
this plugin causes AS(3.5) to be unresponsive

### ISSUE THREAD

**Issue by shingohu** [marius-h/flutter_enhancement_suite#17]
When I edit yaml, the AS gets no response and waits a few minutes before I can continue
My AS version is 3.5

**Comment by marius-h** [comment]
What yaml file were you trying to edit?

**Comment by shingohu** [comment]
pubspec,just input will no response

**Comment by marius-h** [comment]
I tried reproducing your issue on Android Studio 3.5 but I couldn't get it to freeze. Does this happen to every pubspec file which you are trying to edit or is there only a specific one that freezes AS?

**Comment by shingohu** [comment]
yes ,every pubspec file，if I disable or uninstall this plugin， everything is ok

**Comment by marius-h** [comment]
Is it possible that your issue relates to #18 ?

**Comment by shingohu** [comment]
AS(3.5) is still unresponsive ,but update to 3.5.1 everything is ok

**Comment by lizhuoyuan** [comment]
as : 4.0.1
same issue
os: windows
